__tests__/seed.js

```diff
@@ -13,20 +13,24 @@ const knex = require('knex')({
 });
 
 
-
-
-
-
 const apiKey = 'FORTESTFORTESTFORTESTFORTESTFORTEST';
-const entity = {
+const wallet = {
   id: 10,
   name: 'fortest',
-  wallet: 'fortest',
   password: 'test1234',
   passwordHash: '31dd4fe716e1a908f0e9612c1a0e92bfdd9f66e75ae12244b4ee8309d5b869d435182f5848b67177aa17a05f9306e23c10ba41675933e2cb20c66f1b009570c1',
   salt: 'TnDe2LDPS7VaPD9GQWL3fhG4jk194nde',
   type: 'p',
 };
+// const entity = {
```

Comment:
@mzyau why do you delete the entity? It lead to error on my side?

Reply:
Hm what tests are you getting errors in? Maybe it's from a test from the old API? I only set this up for the /auth and /token routes so far. Because we are no longer using entity table, we should no longer reference this entity object. Everything should now be linked to the new wallets object in this file instead. So for example instead of referencing seed.entity.wallet, we should reference seed.wallet.name, and any other entity property should reference the wallet object instead too.